content/articles/building-and-deploying-a-language-detection-system-using-flask/index.md

```diff
@@ -0,0 +1,370 @@
+<link  href='https://fonts.googleapis.com/css?family=Arimo'  rel='stylesheet'  type='text/css'>
+<link  href='https://fonts.googleapis.com/css?family=Hind:300'  rel='stylesheet'  type='text/css'>
+<link  href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300'  rel='stylesheet'  type='text/css'>
+<link  rel="stylesheet"  href="{{ url_for('static', filename='css/style.css') }}">
+
+</head>
+<body>
+<div  class="login">
+<h1>What Language is this?</h1>
+<!-- User Input -->
+<form  action="{{ url_for('predict')}}"method="post">
+<input  type="text"  name="text"  placeholder="Text"  required="required"  />
+<button  type="submit"  class="btn btn-primary btn-block btn-large">Predict</button>
+</form>
+<br>
+<br>
+{{ prediction_text }}
+</div>
+</body>
+</html>
+```
+
+On the `style.css` file, make the following changes to style our webpage.
+
+```css
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+.btn-block { width: 100%; display:block; }
+.btn-large { padding: 8px 13px; font-size: 16px; line-height: normal; -webkit-border-radius: 4px; -moz-border-radius: 5px; border-radius: 5px; }
+.btn-primary.active { color: rgba(241, 241, 241, 0.75); }
+.btn-primary, .btn-primary:hover { text-shadow: 0 -1px 0 rgba(36, 33, 33, 0.25); color: #ffffff; }
+.btn-primary { background-color: #4871c4; background-image: -moz-linear-gradient(top, #67b9e6, #3d6dce); background-image: -ms-linear-gradient(top, #6eb6de, #4a77d4); background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#6eb6de), to(#4a77d4)); background-image: -webkit-linear-gradient(top, #6eb6de, #4a77d4); background-image: -o-linear-gradient(top, #6eb6de, #4a77d4); background-image: linear-gradient(top, #6eb6de, #4a77d4); background-repeat: repeat-x; filter: progid:dximagetransform.microsoft.gradient(startColorstr=#6eb6de, endColorstr=#4a77d4, GradientType=0);  border: 1px solid #3762bc; text-shadow: 1px 1px 1px rgba(0,0,0,0.4); box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.2), 0 1px 2px rgba(0, 0, 0, 0.5); }
+
+
+body {
+	background: #123f3f; 
+	color: #fff;
+	height:100%;
+	width: 100%;
+	letter-spacing:1.2px;
+	font-family: 'Open Sans', sans-serif;
+	font-size: 18px;
+	text-align:center;
+	
+}
+.login { 
+	width:400px;
+	height:400px;
+	position: absolute;
+	margin: -150px 0 0 -150px;
+	top: 40%;
+	left: 50%;
+}
+
+.login h1 { color: #f3f6f4; text-shadow: 0 0 10px rgba(14, 13, 13, 0.3); letter-spacing:1px; text-align:center; }
+
+input { 
+	color: #f3f6f4;
+	background: rgba(0,0,0,0.3);
+	width: 100%; 
+	margin-bottom: 10px; 
+	font-size: 14px;	
+	text-shadow: 1px 1px 1px rgba(0,0,0,0.3);
+	border: none;
+	outline: none;
+	padding: 10px;
+	border: 1px solid rgba(0,0,0,0.3);
+	border-radius: 4px;
+	box-shadow: inset 0 -5px 45px rgba(100,100,100,0.2), 0 1px 1px rgba(255,255,255,0.2);
+}
+input:focus { box-shadow: inset 0 -5px 40px rgba(77, 76, 76, 0.4), 0 1px 1px rgba(240, 237, 237, 0.2); }
```

Comment:
Usually, there is nothing much to explain in CSS unless the whole article is about CSS.

If you're not doing anything significant, I would suggest you to remove a few of them. Instead, add a reference link mentioning the GitHub repo containing the same contents.